src/middleware/auth.js

```diff
@@ -43,7 +43,7 @@ export async function validateAuthentication(req, res, next) {
     })
   }
 
-  const userId = jwt.decode(token)
+  const { userId } = jwt.decode(token)
```

Comment:
should this not be `userId.id`?

Reply:
The token is formed like this:

``     const token = jwt.sign(
        { userId: user.id, email: user.email },
        process.env.JWT_SECRET
      )
      ``
      
So when we destructure it in the auth middleware, we just pull out the userId

Reply:
Ah I get you - hadn't realised it was already destructured at this point.